<ui version="4.0" >
 <class>helpClass</class>
 <widget class="QDialog" name="helpClass" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>help</string>
  </property>
  <widget class="QPushButton" name="pushButton_6" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>720</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>返回</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7" >
   <property name="geometry" >
    <rect>
     <x>380</x>
     <y>720</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>向下</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8" >
   <property name="geometry" >
    <rect>
     <x>530</x>
     <y>720</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text" >
    <string>向上</string>
   </property>
  </widget>
  <widget class="QLabel" name="versionlabel" >
   <property name="geometry" >
    <rect>
     <x>700</x>
     <y>720</y>
     <width>271</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font" >
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="frameShape" >
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow" >
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text" >
    <string>Ver0.01</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11" />
 <resources/>
 <connections/>
</ui>
